src/main/java/seedu/java/util/Ui.java

```diff
@@ -0,0 +1,57 @@
+package seedu.java.util;
+
+import java.util.Scanner;
+
+/**
+ * A UI for user interaction.
+ * intro()
+ * showLoadingError()
+ * inputLoop()
+ */
+public class Ui {
+    String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /**
+     * Prints the logo & gives instructions.
+     */
+    public void intro() {
```

Comment:
Can consider using more descriptive names for this class (i.e. introduceBot, readInput or something)